pip_api/_parse_requirements.py

```diff
@@ -172,6 +177,15 @@ def _url_to_path(url):
     return path
 
 
+class Requirement(requirements.Requirement):
```

Comment:
Can we import these at the module level? At the moment I'm importing from `pip_api._parse_requirements` in order to write proper type hints.

Reply:
Yeah, great call. I'll make that change.

Reply:
https://github.com/di/pip-api/pull/126/commits/a5aaa4d071718e8ababe557bf8370ae0cf61d8bd